src/components/aws/accounts/ListComponent.js

```diff
@@ -130,6 +141,7 @@ Item.propTypes = {
       PropTypes.shape({
         error: PropTypes.string.isRequired,
         nextPending: PropTypes.bool.isRequired,
+        accountOwner: PropTypes.bool.isRequired,
```

Comment:
This should not be a required props.
If the account is not shared the accountOwner prop will be undefined.